### PR TITLE
Feat: Badges Component

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Added Badges Component For All the types: Badges on text, Badge on Icon, Badge on button and badge on avatar

### CODE REVIEW

File: utils/badge.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+@import url('./_colors.css');
```
Comment (JantuDeb) [inline utils/badge.css]: You can import color.css in base css, you don't need to import it separately for badge or other components.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes.

File: utils/badge.css (hunk 1 above)
Comment (munjalSaksham71) [reply utils/badge.css]: Can you please check now

**Review by sks-21 — APPROVED** [review]
Good to merge

**Pull request merged**